man/systemd-journald.service.xml

```diff
@@ -218,6 +218,20 @@ systemd-tmpfiles --create --prefix /var/log/journal</programlisting>
 
         <xi:include href="version-info.xml" xpointer="v228"/></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term>SIGHUP</term>
+
+        <listitem><para>Upon reception of the <constant>SIGHUP</constant> process signal
+         <command>systemd-journald</command> will reload its configuration values
+         and update the kernel log buffer and journals to reflect the new configuration.
+         The kernel log buffer will be flushed and udpated as part of the reload if
+         <command>ReadKmsg</command> has changed. For journal, volatile log data will be flushed
+         to disk if present or archived in memory. Similar to restart, journald will initially use
+         volatile storage upon reload until a call to <command>journalctl --flush</command> is made.</para>
```

Comment:
>  Similar to restart, journald will initially use volatile storage upon reload until a call to <command>journalctl --flush</command> is made.

Why?? I do not think we should do that on reload...

Reply:
Yeah the storage location shouldn't be changed on reload